config.xml

```diff
@@ -3,7 +3,7 @@
     <name>Beanconqueror</name>
     <description />
     <author email="lars.saalbach@googlemail.com" href="https://github.com/graphefruit/beanconqueror">Lars Saalbach</author>
-    <content src="index.html" />
+    <content original-src="index.html" src="http://localhost:8100" />
```

Comment:
Was this done from ide?
Else I'd revert it likely, because I don't know if there are any kind of problems on ios/android later on

Reply:
No idea. I haven't used Ionic for 3-4 years, but I believe this is due to some ionic command